data/languages/uPD77016.slaspec

```diff
@@ -205,16 +205,26 @@ attach variables [ rege  regxe  regye                                  ] [ R0E
 attach variables [ regeh regxeh regyeh                                 ] [ R0EH R1EH R2EH R3EH R4EH R5EH R6EH R7EH ];
 attach variables [                                              top_hl ] [ R0HL R1HL R2HL R3HL R4HL R5HL R6HL R7HL ];
 
-# Note: sr the following registers are labeled as unconditional and only appear in A-5.
-# A-6 gives an extra bit for dest1, but that would require 32 underscores here, so we ignore that.
-# lc is labeled as source2 only, as well.
+# Note: From disassembly it looks like the A-5 "conditional" registers are the regular pointer registers
+# and the "unconditional" registers are the system ones.  This means that bit 22 can be considered part of
+# the register (as sort of implied by the dotted line in A-5 and A-6), and it's just that system registers
+# do not allow conditional moves.  Of course any "conditional" instruction can be made unconditional by
+# setting the conditional clause to IF (TRUE).
+# Also that set immediate in A-6 probably doesn't allow set immediate on the system registers.
+# DMY is reg 0x24, not 0x11, which makes sense as Y memory is associated with DP4-7/DN4-7.
+# Order of the system registers still unclear.  My binary only includes 0x20, 0x21, 0x22 and 0x26.
+# From context 0x20 is obviously SR, 0x21 probably EIR, and 0x22 probably STK. 0x26 is unclear.
```

Comment:
It might be useful to clarify what "my binary", is. (For Wii Speak, I only see 0x20 and 0x21, in contexts where 0x20 looks like SR and 0x21 looks like EIR.)

Reply:
Firmware from RNSE model 192 SD card and MP3 decoder DSP.
A NEC UPD77115A, although it seems to have the MP3 decode baked into instruction ROM which isn’t mentioned anywhere in the datasheets.